recipes/readline/all/conanfile.py

```diff
@@ -41,55 +39,48 @@ def requirements(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        del self.settings.compiler.libcxx
```

Comment:
why is it removed?

Reply:
I'm currently having to debug this on a Windows machine and the default profile generated by conan 2 doesn't have that option and thus throws an error when exporting. Forgot that the profile generated on unix platforms is different

Reply:
just replace by `self.settings.rm_safe("compiler.libcxx")`